app/assets/stylesheets/components/_external-link.scss

```diff
@@ -0,0 +1,7 @@
+.usa-link--external {
```

Comment:
There may be a better way than inline code comment, but I feel we should try to capture what we settled on in our direct conversations around why we're overriding these styles in this way. That is: We're not (yet) using USWDS links consistently, where we'd expect color changes on hover, so in order to prevent the icon from flipping color independently from the rest of the link text on hover, we have to override it.

In the future, we'll want to either apply `usa-link` to all links, or to [configure `$theme-global-link-styles`](https://github.com/uswds/uswds/blob/7fede9d3531ecc4b073fe4a3115f5a2b001f94cc/src/stylesheets/global/_typography.scss#L17-L19) to apply link styling globally. This will bring hover color styles, at which point we can remove these overrides.

And now that I've written it down, maybe this can suffice for capturing 😄